libraries/AP_HAL/utility/functor.h

```diff
@@ -15,10 +15,9 @@
  * You should have received a copy of the GNU General Public License along
  * with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-
 #pragma once
-#ifndef __FUNCTOR_H__
-#define __FUNCTOR_H__
+
+#include <type_traits>
```

Comment:
clang certainly has it.... maybe not the one used in QURT.

So, it's not time yet to do this change. I'll remove this patch.